src/mobile-services.js

```diff
@@ -2,7 +2,11 @@ let config = {
   "version": 1,
   "namespace": "voyager",
   "clientId": "voyager-ionic-example",
-  "services": []
+  "services": [
+    {
```

Comment:
Why is there now an empty object inside the services array?

Reply:
We don't populate this as it would be different for each openshift instance

Reply:
got it typo